gaek/ndb_json.py

```diff
@@ -67,14 +67,21 @@ def encode_generator(obj):
   return list(obj)
 
 
-def encode_key(obj):
+def encode_key_as_entity(obj):
```

Comment:
Would you mind keeping `encode_key` and pointing it to this function to maintain compatibility in case any users are calling it directly?